app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,7 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="selectedColor">#FF000000</color>
+    <color name="mode">#403D3D</color>
+    <color name="mode1">#E3E0E0</color>
```

Comment:
Lepiej byłoby to czytelniej podpisać np. jako lightMode i darkMode, tak żeby potem szybciej to ogarnąć. Chyba, że źle zrozumiałem i to nie dotyczy trybu ciemnego.